src/org/zaproxy/zap/extension/params/ParamsPanel.java

```diff
@@ -204,23 +204,26 @@ private JScrollPane getJScrollPane() {
 	
 	private void setParamsTableColumnSizes() {
 		
-		paramsTable.getColumnModel().getColumn(0).setMinWidth(50);
-		paramsTable.getColumnModel().getColumn(0).setPreferredWidth(100);	// type
+		paramsTable.getColumnModel().getColumn(0).setMinWidth(60);
+		paramsTable.getColumnModel().getColumn(0).setPreferredWidth(60);	// type
 		
 		paramsTable.getColumnModel().getColumn(1).setMinWidth(100);
-		paramsTable.getColumnModel().getColumn(1).setPreferredWidth(200);	// name
+		paramsTable.getColumnModel().getColumn(1).setPreferredWidth(100);	// name
 		
-		paramsTable.getColumnModel().getColumn(2).setMinWidth(50);
-		paramsTable.getColumnModel().getColumn(2).setPreferredWidth(100);	// used
+		paramsTable.getColumnModel().getColumn(2).setMinWidth(40);
+		paramsTable.getColumnModel().getColumn(2).setPreferredWidth(40);	// used
 		
-		paramsTable.getColumnModel().getColumn(3).setMinWidth(50);
-		paramsTable.getColumnModel().getColumn(3).setPreferredWidth(100);	// numvals
+		paramsTable.getColumnModel().getColumn(3).setMinWidth(40);
+		paramsTable.getColumnModel().getColumn(3).setPreferredWidth(40);	// numvals

```

Comment:
This hides part of the column name even if there's space, was that on purpose? (First time users would have to resize the column to know what values the column has.)